templates/organization/organization_position_list.html

```diff
@@ -37,8 +55,16 @@ <h1>{{ organization.organization_name }}: Position List</h1>
                class="u-no-break"><span class="glyphicon glyphicon-new-window"></span></a>
             {% endif %}
         </td>
+         {% endif %}
```

Comment:
It looks like you have an extra `endif` here? I would recommend removing the `if organization.organization_website` and both `endif` statements.